### DISCUSSION TITLE
如图我的图名称为两行，第一行为图的名称，第二行为其中子图的名称，我只想将其中第一行放入到图目录中

### DISCUSSION THREAD

**Discussion by RingKing1** [TheNetAdmin/zjuthesis#336] [General]
![image](https://github.com/TheNetAdmin/zjuthesis/assets/45783443/5a62110a-72bc-4697-bc81-ecdc3e1d0c08)
![image](https://github.com/TheNetAdmin/zjuthesis/assets/45783443/6bb232ab-0460-4062-b23a-0b892bc79e25)
其中图目录应该引用的没有冒号

**Comment by note286** [comment]
```latex
\caption[出现在目录]{出现在正文}
```

**Reply by RingKing1** [reply]
谢谢解答，我也刚刚发现了

**Comment by RingKing1** [comment]
`\begin{figure}[htbp]
    \centering
    \includegraphics[width=.3\linewidth]{example-image-a}
    \caption[装置示意图]{\label{fig:fig-placeholder}装置示意图：\\ (a) 正视图；(b) 俯视图}
\end{figure}
`
问题解决了，我将方法贴在这里
